src/journal/bsod.c

```diff
@@ -222,7 +222,7 @@ static int display_emergency_message_fullscreen(const char *message) {
         if (r < 0)
                 log_warning_errno(r, "Failed to move terminal cursor position, ignoring: %m");
 
-        r = loop_write(fd, "Press any key to exit...", SIZE_MAX);
+        r = loop_write(fd, ANSI_BACKGROUND_BLUE "Press any key to exit...", SIZE_MAX);
```

Comment:
Shouldn't this be `ansi_background_blue()` (i.e. conditional)?

Reply:
well, the tool is pretty useless if we don't have ANSI colors or cursor positioning or block drawing chars... 

We probably should add some guards to make it a nop on dumb terminals. but let's do that in some other PR